### ISSUE TITLE
Backup to Rclone without creating tmp files

### ISSUE THREAD

**Issue by Lexxior** [djmaze/resticker#161]
I have a NAS that I have mounted in the docker container on my server
If I run the container restic creates tmp files until the storage on the server is full and then cancels the backup, leaving the tmp files

If I run restic bare metal on the server with `restic -r rclone:Onedrive:/Backup/NAS backup  /volumes/NAS --verbose` it backs up directly to my Onedrive, not creating any tmp files and filling up the storage

Is it possible to achieve this behavior in the docker container too?

**Comment by pquerner** [comment]
How is the NAS mounted in the docker container? As a volume? Does mounting it as [read-only](https://docs.docker.com/storage/volumes/#use-a-read-only-volume) help?

**Comment by Lexxior** [comment]
Yes as a volume via NFS - read only makes no difference and the temp files get created on the docker host, not on the NAS

**Comment by djmaze** [comment]
AFAIK restic always creates temp files during backup and recovery. You can only [change the path](https://restic.readthedocs.io/en/latest/manual_rest.html#temporary-files).
